src/main/java/amogus/Amogus.java

```diff
@@ -0,0 +1,55 @@
+package amogus;
+
+import command.Command;
+import tasks.TaskList;
+
+import java.io.IOException;
+
+public class Amogus {
+
+    static final String NAME = "amogus";
+    private static final String path = "./data/Tasks.TaskList.txt";
+
+    /**
+     * Creates all necessary objects for interaction between each other,
+     * namely Amogus.UI for responses to user, Amogus.FileStorage for hard disk load
+     * and store, Tasks.TaskList for list of tasks.
+     */
+    public static void Chat() {
+        UI ui = new UI();
+        ui.welcome();
+        FileStorage fs = new FileStorage(path);
```

Comment:
Could use a more descriptive name here so it is more apparent in subsequent code what this variable is referring to
```suggestion
        FileStorage fileStorage = new FileStorage(path);
```